packages/flutter/test/material/chip_theme_test.dart

```diff
@@ -271,10 +371,11 @@ void main() {
 
     final RenderBox materialBox = getMaterialBox(tester);
     expect(materialBox, paints..circle(color: backgroundColor));
-    expect(tester.getSize(find.byType(RawChip)), const Size(250, 250)); // label + padding + labelPadding
+    expect(tester.getSize(find.byType(RawChip)), const Size(400, 250)); // icon + label + padding + labelPadding
     expect(getMaterial(tester).elevation, elevation);
     expect(getMaterial(tester).shape, shape);
     expect(getLabelStyle(tester).style.fontSize, 32);
+    expect(getIconTheme(tester), customIconTheme, reason:'Chip Icon Theme does not reflect custom iconTheme.');
```

Comment:
This is not necessary. 
```suggestion
    expect(getIconTheme(tester), customIconTheme);
```